impl/src/main/java/org/jboss/weld/event/NotificationOptionKeys.java

```diff
@@ -29,6 +29,7 @@
 public final class NotificationOptionKeys {
 
     public static final String MODE = "org.jboss.weld.notification.mode";
+    public static final String TIMEOUT = "org.jboss.weld.notifications.timeout";
```

Comment:
Minor - we should use unified name - probably `org.jboss.weld.notification.timeout`